cip.go

```diff
@@ -123,6 +123,10 @@ func main() {
 		"audit-gcp-project-id",
 		os.Getenv("CIP_AUDIT_GCP_PROJECT_ID"),
 		"GCP project ID (name); used for labeling error reporting logs to GCP")
+	maxImageSizePtr := flag.Float64(
+		"max-image-size",
+		2,
```

Comment:
I would just use MiB instead of GiB. It is more granular with no loss of functionaliy.